pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -330,6 +330,7 @@ in with passthru; stdenv.mkDerivation {
   postInstall = let
     # References *not* to nuke from (sys)config files
     keep-references = concatMapStringsSep " " (val: "-e ${val}") ([
+      "${placeholder "out"}"
```

Comment:
```suggestion
      (placeholder "out")
```

The result for placeholder is already a string.

Reply:
Tested this locally and it is a no op. I am waiting for ofborg, then pushing the change and then merging.